joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -44,6 +44,8 @@ class AstCreator(
       .map(_.stripPrefix(java.io.File.separator))
       .getOrElse(fileName)
 
+  private val internalLineAndColNum: Option[Integer] = Option(1)
```

Comment:
I'd feel more comfortable if these were fresh values for each node
```suggestion
  private def internalLineAndColNum: Option[Integer] = Option(1)
```

Reply:
But if this is what Pythonsrc does then we can leave it at `val`